card/lib/card/query/card_query/match_attributes.rb

```diff
@@ -49,9 +49,9 @@ def field_match field, val, cxn
         end
 
         # TODO: move sql to SqlStatement
-        def key_like pattern, no_junction=false
+        def key_like pattern, junction_ok=true
```

Comment:
This retains the default behavior and fixes the incorrect handling of `complete`